packages/sdk/src/conversation/sso/defaultBotSsoExecutionActivityHandler.browser.ts

```diff
@@ -1,6 +1,7 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
+/* eslint-disable @typescript-eslint/require-await */
```

Comment:
Would directly removing all `async` be better? Since all methods are just throwing exception, no side-effect if removing `async`?

Reply:
Anyway, also no side-effect to disable the rule since bot is server-side sdk.

Reply:
Yes, this file is running inside a browser environment (not supported) and is used to align the function signature.